railties/lib/rails/generators/rails/app/templates/Dockerfile.tt

```diff
@@ -78,6 +78,10 @@ RUN SECRET_KEY_BASE_DUMMY=1 ./bin/rails assets:precompile
 
 <% end -%>
 
+<% if using_node? -%>
+RUN rm -rf node_modules
+<% end %>
```

Comment:
can you move this under line 88?

Reply:
Thank you for the feedback. If I'm not mistaken, `node_module` does not exist in line 88. It is copied at line 93.